modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/DhsClientHttp.scala

```diff
@@ -137,10 +139,11 @@ class DhsClientSim(date: LocalDate) extends DhsClient {
       TrySeq(f"S${date.format(format)}S${counter.incrementAndGet()}%04d")
     })
 
-  override def setKeywords(id: ImageFileId, keywords: KeywordBag, finalFlag: Boolean): SeqAction[Unit] = SeqAction(())
+  override def setKeywords(id: ImageFileId, keywords: KeywordBag, finalFlag: Boolean): SeqAction[Unit] = EitherT(Task(Log.info(keywords.keywords.map(k => s"${k.name} = ${k.value}").mkString(", ")).right))
```

Comment:
The PR talks about a fix to FITS keywords sent on the wrong order but I only see a logging change. Where are the keywords reverted?